src/main/scala/circt/stage/ChiselStage.scala

```diff
@@ -96,14 +106,18 @@ object ChiselStage {
       CIRCTTargetAnnotation(CIRCTTarget.CHIRRTL)
     ) ++ (new BareShell("circt") with CLI).parse(args)
 
-    phase
-      .transform(annos)
-      .collectFirst {
-        case a: ChiselCircuitAnnotation => CircuitSerializationAnnotation(a.circuit, "", FirrtlFileFormat).getBytes
-      }
-      .get
-      .map(_.toChar)
-      .mkString
+    val resultAnnos = phase.transform(annos)
+
+    var circuitAnno: Option[CircuitSerializationAnnotation] = None
```

Comment:
For your consideration: This is the type of thing that the [package view/options](https://github.com/chipsalliance/chisel/blob/main/src/main/scala/circt/stage/package.scala) is supposed to handle for you. It could be used here or not. I want to gut that and the rest of stage/phase so I can see not wanting to use it here.

Reply:
I could move it in there if you think it's better. I don't have a strong opinion as this logic is somewhat duplicative of what's in WriteOutputAnnotations but it would be very difficult to share the code between the two. I figure neither this code nor WriteOutputAnnotations are long for this world so maybe it's not worth worrying about since we'll just rewrite it soon 🤷‍♀️